src/styles/index.scss

```diff
@@ -60,9 +103,30 @@ h6 {
   @apply text-primary;
 }
 
+h1, h2 {
+  font-family: Agrandir-Wide, sans-serif;
+  @apply font-bold
+}
+
+h3,
```

Comment:
**suggestion**: i think we should just "reset" heading styles and style each heading element inline.

https://meyerweb.com/eric/tools/css/reset/